base_tier_validation/__manifest__.py

```diff
@@ -1,26 +1,29 @@
 # Copyright 2017-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2019 Ecosoft Co., Ltd. (http://ecosoft.co.th)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
 {
-    "name": "Base Tier Validation",
```

Comment:
@ps-tubtim when do the [MIG] or [IMP], we do not change the already existing code if it is not an improvement or bug fix.
(But it is our new addon, we can format whatever we want :)